manifests/fpm/daemon.pp

```diff
@@ -16,30 +16,6 @@
   $log_dir_mode = '0770'
 ) {
 
-  # Hack-ish to default to user for group too
-  $log_group_final = $log_group ? {
-    false   => $log_owner,
-    default => $log_group,
-  }
-
-  if ($ensure == 'present') {
-
-    service { 'php5-fpm':
-      ensure    => running,
-      enable    => true,
-      restart   => 'service php5-fpm reload',
-      hasstatus => true,
-      require   => Package['php5-fpm'],
-    }
-
-    file { '/etc/php5/fpm/php-fpm.conf':
-      notify  => Service['php5-fpm'],
-      content => template('php/fpm/php-fpm.conf.erb'),
-      owner   => root,
-      group   => root,
-      mode    => '0644',
-    }
-
-  }
+  warn('php::fpm::daemon is being deprecated, please move your php-fpm.conf settings into php::fpm..')
```

Comment:
heh. i just realized: if we leave a hollow shell, it's not a warn, it's actually a `fail`, it's not "being deprecated", it *is* deprecated.

this will, however, require a major version bump. but then again, we're still in 0.9.0, so all bets are off ;)